zookeeper.logrotate

```diff
@@ -1,11 +1,9 @@
 /var/log/zookeeper/*.log {
-       weekly
-       rotate 4
-       maxsize 100M
-       minsize 10M
-       copytruncate
-       delaycompress
-       compress
-       notifempty
-       missingok
+    weekly
+    rotate 4
+    maxsize 100M
+    minsize 10M
+    copytruncate
+    compress
+    missingok
```

Comment:
There is a minsize set, so notifempty becomes irrelevant. As for delaycompress, it's only really useful when you think the log file will still be written to for a short file after being rotated, and I don't think that can ever be the case with copytruncate.